ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1114,14 +1306,17 @@ export const RuntimePanel = fp.flow(
                   onChange={(value) => setSelectedMachine(value)}
                   validMachineTypes={validMainMachineTypes}
                   machineType={machineName}/>
-                <DiskSizeSelector
-                    idPrefix='runtime'
-                    selectedDiskSize={selectedDiskSize}
-                    onChange={(value) => {
-                      setSelectedDiskSize(value);
-                    }}
-                    disabled={disableControls}
-                    diskSize={diskSize}/>
+                {(runtimeExists && !pdExists) ?
+                    <DiskSizeSelector
```

Comment:
what is the difference between this one with PersistentDiskSizeSelector below?

Reply:
It's different from the local disk selector in UI style and description